css-positioning_absolute-no-advanced-requirements-1/css/styles.css

```diff
@@ -0,0 +1,93 @@
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: system-ui;
+}
+
+main {
+  width: 500px;
+  height: 600px;
+  border: 1px solid black;
+  margin: 5em auto;
+  position: relative;
+}
+
+.box {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 80px;
+  width: 80px;
+  text-align: center;
+}
+
+/**
+ * Look at the selectors defined below.
+ * Position each element, so that the placement matches the name of the selector.
+ * Adjust the rule-sets for each selector to satisfy the name.
+ *
+ * @example
+ * .box--top-left -> This element should be be placed in the top, left corner.
+*/
+
+.box--top-left {
+  background-color: #aed6f1;
+ position: absolute;
+  top: 0;
+  left: 0;
+}
+
+.box--top-center {
+  background-color: #85c1e9;
+  position: relative;
+  margin-left: auto;
+  margin-right: auto;
+  top: 0;
+}
+
+.box--top-right {
+  background-color: #5dade2;
+margin: 0;
+  position: absolute;
+  top: 0;
+  right: 0;
+}
+
+.box--center {
+  background-color: #3498db;
+  margin: auto;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  -ms-transform: translate(-50%, -50%);
```

Comment:
I am not sure what this does :)

Reply:
This center the box which is called center